apps/sailfish/qml/pages/BrowserPage.qml

```diff
@@ -157,6 +157,9 @@ Page {
                         if(browserPage.model.hasDetails()) {
                             browserPage.model.fetchItemDetails(filterModel.mapToSourceIndex(listView.currentIndex));
                             drawer.open = true;
+                        } else {
+                            // no info, but we still want to be able to play/queue
+                            drawer.open = true;
```

Comment:
Yes I intentionally left this in for readability and to indicate it still is a special case. The files model is AFAICS the only model where the drawer is disabled.